src/Neo/SmartContract/Native/TrimmedBlock.cs

```diff
@@ -125,9 +125,14 @@ StackItem IInteroperable.ToStackItem(IReferenceCounter referenceCounter)
                 Header.PrimaryIndex,
                 Header.NextConsensus.ToArray(),
 
-                // Block properties
-                Hashes.Length
-            ]);
+            };
+            if (Header.Version == (uint)BlockVersion.One)
```

Comment:
If we add them at the end, we don't break a smart contract that use Hashes.Length in previous version